pkg/netpol/connlist/exposure_map.go

```diff
@@ -43,7 +43,7 @@ func (ex *exposureMaps) addPeerUnprotectedData(pe *eval.PolicyEngine, peer Peer,
 		return false, err
 	}
 	if !isProtected {
-		ex.addNewEntry(peer, isProtected, isIngress)
+		ex.checkExistanceAndAddNewEntry(peer, isProtected, isIngress)
```

Comment:
I would keep the name simple, just `addNewEntry`

Reply:
done